### DISCUSSION TITLE
Is this project still active?

### DISCUSSION THREAD

**Discussion by KonradLinkowski** [marblejs/marble#410] [General]
I've seen that there was a new version 10 months ago but other than that everything seems dead. What happened?

**Comment by jan-kacina** [comment]
Very good question! My startup project is heavily dependent on MarbleJS (awesome FRP framework, btw.)
It works fine as it is but I'd appreciate any further development, mainly in testing and messaging area...

**Comment by ed3899** [comment]
Same here, this has the potential to rival big contenders such as NestJS.

With an improved API that bases itself solely on immutability.

One of the great downsides of NestJS is that up to a certain extent, it is still mutable. For example a Guard, can must mutate the request if it was to append extra information. 

And heavily reliant on Decorators, which may not be fully type safe when composing multiple of them

**Comment by jan-kacina** [comment]
@JozefFlakus could you provide at least simple `Yes` (it's alive), `No` (it's dead) or `Maybe` (it's a zombie) answer, please?

**Comment by JozefFlakus** [comment]
I would say this project is `dead` or `zombie`. I decided to stop working actively on the further development because I don't see any  future of doing it fully on my own, without any contributors help and simply... pro bono. I don't have any profit, no support and time for doing that. I'm glad to see that someone is using it but, again... I don't want to build things that there is almost zero support from the contributors side. Maybe in the future I'll go back to it when the situation for open-source developers will change but ATM I would like to focus on things that bring me joy. I hope you all understand it.

**Reply by jan-kacina** [reply]
Thanks for the response.

I'd like to be a contributor but even I'm using MarbleJS fourth year, the functional concept MarbleJS's is based on (`io-ts`, `fp-ts`) is beyond my understanding's horizon :)

However, BIG thanks for your work on MarbleJS and I hope you will find time and will to continue in the future!

PS: Currently I'm not in such situation, but hypothetically - if there will be a funding to support your work, would you consider to continue with MarbleJS?

**Reply by JozefFlakus** [reply]
> However, BIG thanks for your work on MarbleJS and I hope you will find time and will to continue in the future

I hope too!

> [...] if there will be a funding to support your work, would you consider to continue with MarbleJS?

I think so. It is really hard to share a full time job together with other passions. By having a founder I would have a possibility to work part-time on the project and continue working on it.
